force-app/main/default/classes/SOQL.cls

```diff
@@ -50,9 +54,23 @@ public with sharing class SOQL {
      * @return A list of Contact Ids.
      * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Id> getContactIdsByLastName (List<String> contactsToQuery) {
+        List<String> filterBy = new List<String>();
+            for(String contactLN : contactsToQuery){
+                if(contactLN == 'Smith' || contactLN == 'Jones'){
+                    filterBy.add(contactLN);
+                }
+            }
```

Comment:
Why would we make a new List<String> if the existing List<String> is already contains the LastNames we want?